### DISCUSSION TITLE
Write Prompts to PNG or to Single File.

### DISCUSSION THREAD

**Discussion by vdruts** [hlky/stable-diffusion-webui#967] [Feature suggestions]
There used to be a section where we could tick to write all prompts/seed info to a single file, now it's missing. Can we please bring it back? Also, can we create an option similar to Automatic1111's where we write the info into the PNG?

**Comment by Michoko92** [comment]
The feature of writing info to PNG chunks can be activated by calling webui.py with the "--save-metadata" parameter. You'll then need a PNG parser like this one to see them: https://www.nayuki.io/page/png-file-chunk-inspector

**Comment by codedealer** [comment]
@bbecausereasonss Update and check the advanced tab. It's present on `dev` at least.
![image](https://user-images.githubusercontent.com/4258136/189506594-5417f43a-636e-4581-aa55-9b3e2f3c8cb0.png)
